Backend/BikesRentalServer.Tests/UsersService/BlockUser.cs

```diff
@@ -0,0 +1,107 @@
+﻿using BikesRentalServer.DataAccess;
+using BikesRentalServer.Models;
+using BikesRentalServer.Services;
+using BikesRentalServer.Tests.Mock;
+using FluentAssertions;
+using Xunit;
+
+namespace BikesRentalServer.Tests.UsersService
+{
+    public class BlockUser
+    {
+        private readonly DatabaseContext _dbContext;
+        private readonly Services.UsersService _usersService;
+
+        public BlockUser()
+        {
+            _dbContext = MockedDbFactory.GetContext();
+            _usersService = new Services.UsersService(_dbContext);
+        }
+
+        [Fact]
+        public void BlockUserThatDoesntExist()
+        {
+            var response = _usersService.BlockUser("");
+
+            response.Status.Should().Be(Status.EntityNotFound);
+            response.Object.Should().Be(null);
+        }
+
+        [Fact]
+        public void BlockUserAlreadyBlocked()
+        {
+            int testId = 100;
+            _dbContext.Users.Add(new User
+            {
+                Id = testId,
+                State = UserState.Banned,
+            });
+            _dbContext.SaveChanges();
+            var response = _usersService.BlockUser(testId.ToString());
+
+            response.Status.Should().Be(Status.InvalidState);
+            response.Object.Should().Be(null);
+        }
+
+        [Fact]
+        public void BlockUserSimpleSuccess()
+        {
+            int testId = 100;
+            _dbContext.Users.Add(new User
+            {
+                Id = testId,
+                State = UserState.Active,
+            });
+            _dbContext.SaveChanges();
+            var response = _usersService.BlockUser(testId.ToString());
```

Comment:
Odseparowałbym tę linię od poprzedniej jedną pustą linią, żeby podkreślić ten podział o którym Piotrek kiedyś wspominał (Arrange, Act, Assert o ile pamiętam). To się pojawia w wielu testach więc nie będę tam już powtarzał